security-acceptance-criteria.md

```diff
@@ -0,0 +1,139 @@
+---
+name: FOSS Security Release XX.XX
+about: Use this template to track all Security-relevant topics for your component with regards to the upcoming Milestone.
+title: "[FOSS NAME] Release Security XX.XX"
+labels: foss, "security analysis"
+team: SIG-Security
+---
+
+<!-- 
+Thanks for your contribution! Please fill out this template as good as possible. 
+Important: Contributing Guidelines can be found here: https://eclipse-tractusx.github.io/docs/oss/how-to-contribute
+Checkout the repository README for process description. 
+-->
+
+# Release Security XX.XX
+
+Source in Catena-X Confluence and Expert Contacts [here](https://confluence.catena-x.net/display/PL/Consortia+Quality+Gates)<br>
```

Comment:
We cannot have links to closed systems, so please move relevant descriptions that are hidden there to a markdown or similar here in this repo

Reply:
I have removed this file following your review showing me the existing template.